### ISSUE TITLE
Decks with several subdecks cannot be imported with AnkiDroid

### ISSUE THREAD

**Issue by paternal** [AnonymerNiklasistanonym/Md2Anki#21]
Hello,
I have the following .md file:

~~~md
# MWE (1509722053)

## Subdeck: Foo (2767618514)

### 1+1=? (60d92eff-fb3b-42ca-81c3-b7ee3f914a85)

2

## Subdeck: Bar (2299577195)

### 1x1=? (b6848e38-ace4-4a13-b85c-9e6c519d3500)

1
~~~

I generate the anki deck:

~~~bash
$ md2anki --version
md2anki 3.0.14b
$ md2anki deck.md
[everything went well: nothing displayed]
~~~

I get the following apkg file: [deck.zip](https://github.com/AnonymerNiklasistanonym/Md2Anki/files/12160093/deck.zip) (I changed the extension to zip because github would not allow me to upload an apkg file).

- This file can be imported with Anki: ✔️
- This file cannot be imported with AnkiDroid ❌

Indeed, when I try to import it with AnkiDroid:

- I first get this loading screen (so far so good):

  ![Capture d’écran du 2023-07-25 12-29-36](https://github.com/AnonymerNiklasistanonym/Md2Anki/assets/6587757/5dfbae9e-7753-488c-9c9b-f248e15c6439)

  *Translation: Importing / Importing notes: 0% / Importing cards: 0% / Post-processing: 0%*

- Then AnkiDroid crashes:

  ![Capture d’écran du 2023-07-25 12-29-45](https://github.com/AnonymerNiklasistanonym/Md2Anki/assets/6587757/e64e5d66-5c10-45e1-b0fe-1378121d60b7)

I do not know if this is a md2anki bug or an AnkiDroid bug. I am reporting bugs on both projects.

Thanks for your work!
Louis

**Comment by paternal** [comment]
Here is the corresponding AnkiDroid issue: https://github.com/ankidroid/Anki-Android/issues/14154

**Comment by paternal** [comment]
Fixed with AnkiDroid version 2.16.1: see ankidroid/Anki-Android#14154
